name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/lists/FollowUpWidget.java

```diff
@@ -133,13 +143,20 @@ protected Composite createItemControl(Composite parent, FollowUpItem item)
 
         Label name = createLabel(composite, item.getSecurity().getName(getClient().getSecurityNameConfig()));
 
-        Label date = createLabel(composite, item.type.getName() + ": " + Values.Date.format(item.date)); //$NON-NLS-1$
-
         composite.addMouseListener(mouseUpAdapter);
         name.addMouseListener(mouseUpAdapter);
-        date.addMouseListener(mouseUpAdapter);
 
-        FormDataFactory.startingWith(logo).thenRight(name).right(new FormAttachment(100)).thenBelow(date);
+        if (!item.date.equals(this.lastDate) || !item.type.equals(this.lastAttrType))
+        {
+            Label date = createLabel(composite, item.type.getName() + ": " + Values.Date.format(item.date)); //$NON-NLS-1$
+            FormDataFactory.startingWith(date).thenBelow(logo).thenRight(name).right(new FormAttachment(100));
+            this.lastDate = item.date;
+            this.lastAttrType = item.type;
```

Comment:
In your example, if there is a refresh, wouldn't it show no headline at all?
There are two items that are collated into one. In the next rundown the "lastDate" and "lastAttrType" match also the first one.

I propose to not use a member variable. For reasons of simplicity, I added a "previous" item to the method.

Reply:
> In your example, if there is a refresh, wouldn't it show no headline at all?

Yeah, apparently it required to override update() method to re-initialize those vars.

> I propose to not use a member variable.

Sure, if there's an easy way to avoid storing state, it's good idea. Thanks for the refactor!